lib/core-server/src/core-presets.test.ts

```diff
@@ -9,9 +9,13 @@ import { executor as managerExecutor } from './manager/builder';
 import { buildDevStandalone } from './build-dev';
 import { buildStaticStandalone } from './build-static';
 
+// nx-ignore-next-line
 import reactOptions from '../../../app/react/src/server/options';
+// nx-ignore-next-line
 import vue3Options from '../../../app/vue3/src/server/options';
+// nx-ignore-next-line
 import htmlOptions from '../../../app/html/src/server/options';
+// nx-ignore-next-line
```

Comment:
@shilman If we wanted to remove these circular dependencies we could extract this into it's own util, or duplicate the code (wouldn't recommend that)